### PR TITLE
Export Maya workspace path.

### PR DESCRIPTION
**Pull request by jerome-jacob**

### CODE REVIEW

File: COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp [hunk 1]
```diff
@@ -305,6 +305,28 @@ namespace COLLADAMaya
...
+						mStreamWriter.appendAttribute(COLLADASW::CSWC::CSW_ATTRIBUTE_PROFILE, PROFILE_MAYA);
+						{
+							mStreamWriter.openElement("workspace");
+							mStreamWriter.appendText(workspaceURI.getURIString());
+							mStreamWriter.closeElement();
+						}
+						mStreamWriter.closeElement();
+					}
+					mStreamWriter.closeElement();
+				}
```
Comment (hubert-sarret-sb) [inline COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp]: Display an error message, as it is it is failing to add workspace base URI silently

File: COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp [hunk 2]
```diff
@@ -305,6 +305,28 @@ namespace COLLADAMaya
...
+			}
```
Comment (hubert-sarret-sb) [inline COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp]: Same

File: COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp (hunk 1 above)
Comment (jerome-jacob) [inline COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp]: Warnings added.

**Comment by hubert-sarret-sb** [comment]
Ok for me @RemiArnaud

File: COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp (hunk 1 above)
Comment (mbarnes-sb) [inline COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp]: @jerome-jacob where is this `extra` element in the document structure? if it is inside `<asset>` then it is invalid. Instead place it in `<COLLADA><extra>` to achieve document scope.

Comment (jerome-jacob) [inline COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp]: It is.

**Pull request merged**